### PR TITLE
[dev] [InfluxDB-Retriever/Storer] Disable validate responses,  fix a wrong param webUtils, remove drop and remove unused configs

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR? (Link to an issue, e.g. #99999)** 

* **Other information**:

### CODE REVIEW

File: storage/time-series/influxdb/retriever/app/express/index.js [hunk 1]
```diff
@@ -73,17 +73,15 @@ module.exports = (routes, serviceState, openApiPath) => {
         stateManager: serviceState,
         logger,
       }),
-      responseCompressInterceptor({
-        supportTrustProxy: configExpress.trustproxy,
-      }),
+      responseCompressInterceptor({}),
```
Comment (thiagodpf) [inline storage/time-series/influxdb/retriever/app/express/index.js:76]: You don't need to pass anything as a parameter that by default an empty object is created

File: storage/time-series/influxdb/retriever/app/express/index.js [hunk 2]
```diff
@@ -73,17 +73,15 @@ module.exports = (routes, serviceState, openApiPath) => {
...
       requestLogInterceptor({
         logger,
       }),
     ],
     routes: (routes).flat(),
     logger,
     config: {
-      trustproxy: configExpress.trustproxy,
+      supportTrustProxy: configExpress.trustproxy,
     },
```
Comment (thiagodpf) [inline storage/time-series/influxdb/retriever/app/express/index.js:85]: The `config` object no longer exists, pass the attribute `supportTrustProxy` directly

https://github.com/dojot/dojot-microservice-sdk-js/blob/master/lib/webUtils/framework/createExpress.js#L88

**Review by thiagodpf — COMMENTED** [review]
Just a few observations :grinning:

**Review by thiagodpf — APPROVED** [review]
:1st_place_medal:

**Review by rascaraficci — APPROVED** [review]
It looks OK!

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**